src/components/date-picker/date-picker--range.html

```diff
@@ -0,0 +1,24 @@
+<!-- Ranged -->
+<div class="bx--form-item">
+  <div data-date-picker data-date-picker-type="range" class="bx--date-picker bx--date-picker--range">
+    <div class="bx--date-picker__container">
```

Comment:
For consistency, should this be `__container` or `-container`? I thought `__` was only used when we are positioning an element relatively? Correct me if I'm wrong